crates/sui-core/src/authority/authority_store.rs

```diff
@@ -1050,6 +1096,40 @@ impl AuthorityStore {
             .cloned()
             .collect();
 
+        let mut output = Vec::new();
+        let _ = deleted
+            .iter()
+            .filter(|(object_id, _)| {
+                let object = objects.get(object_id).unwrap();
```

Comment:
Could deleted contain unwrapped_then_deleted? If so, objects.get() is not guaranteed to return Some

Reply:
I thought I changed this but it got reverted, thanks